babel.config.js

```diff
@@ -10,4 +10,85 @@ module.exports = {
       ],
       '@babel/preset-react',
     ],
-};
+};module.exports = function(api) {
+  var validEnv = ['development', 'test', 'production']
+  var currentEnv = api.env()
+  var isDevelopmentEnv = api.env('development')
+  var isProductionEnv = api.env('production')
+  var isTestEnv = api.env('test')
+
+  if (!validEnv.includes(currentEnv)) {
+    throw new Error(
+      'Please specify a valid `NODE_ENV` or ' +
+        '`BABEL_ENV` environment variables. Valid values are "development", ' +
+        '"test", and "production". Instead, received: ' +
+        JSON.stringify(currentEnv) +
+        '.'
+    )
```

Comment:
- [ ] Please kindly take note that I was unable to access your app until I delete these few lines of code from line 13 to 94; a senior colleague also confirmed the same. If you can refactor this file as suggested below, I think all will be fine. 🙏 
```suggestion
module.exports = {
    presets: [
      [
        '@babel/preset-env',
        {
          targets: {
            node: 'current',
          },
        },
      ],
      '@babel/preset-react',
    ],
};
```

Reply:
Hi @Oyepriscilla .... Your feedback explanation doesn't mean well to me. Reaching out to me to further show you either by a short video or a screenshot or how you could run the application could have solved and saved us this back and forth. This is not acceptable. My app is running and working.